src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -296,11 +312,20 @@ private double altTurnSmooth(double stickAng) {
    *
    * @return the current position of the robot on the field
    */
-  private Pose2d getPose() {
+  public Pose2d getPose() {
     Pose2d pose = m_odometry.getPoseMeters();
     return pose;
   }
 
+  public void moveChassisSpeeds(ChassisSpeeds spds) {
+    SwerveModuleState[] swerveModuleStates =
```

Comment:
Take a look at the sample code and:
1. Consider renaming this method
2. Consider renaming the input to something more meaningful (again, see sample code)
3. Figure out what's different between this implementation and the sample code, and see why that's important.

Reply:
uses local kinematics object now